JSQMessagesViewController/Model/JSQMessageMediaData.h

```diff
@@ -78,4 +78,24 @@
  */
 - (NSUInteger)mediaHash;
 
+@optional
```

Comment:
I think I'd like to target this feature for the 8.0 release, so we can make a breaking change and have this be non-optional instead